core/src/main/web/app/controlpanel/controlpanel-directive.js

```diff
@@ -36,7 +36,7 @@
 
         $scope.gotoControlPanel = function(event) {
           if (bkUtils.isMiddleClick(event)) {
-            window.open("./");
+            window.open("../../");
```

Comment:
bkHelper.serverUrl(x) gives a URL of the form 'ip:port/hash/x', but the click is supposed to redirect to the root ('ip:port/'). Is there another function that returns the base url without the hash?